package.json

```diff
@@ -7,21 +7,21 @@
     "yarn": "1.x.x"
   },
   "scripts": {
-    "acceptance": "sh scripts/acceptance.sh",
-    "acceptance-record": "sh scripts/acceptance_record.sh",
-    "assets": "sh scripts/assets.sh",
+    "acceptance": "scripts/acceptance.sh",
+    "acceptance-record": "scripts/acceptance_record.sh",
+    "assets": "scripts/assets.sh",
     "debug:prod": "NODE_ENV=production node -r dotenv/config --max_old_space_size=1024 ./src --verbose --debugProd",
-    "deploy": "sh scripts/deploy.sh",
-    "mocha": "sh scripts/mocha.sh",
+    "deploy": "scripts/deploy.sh",
+    "mocha": "scripts/mocha.sh",
     "precommit": "lint-staged",
     "prepare": "patch-package",
     "prettier-write": "yarn run prettier --write",
     "relay": "node scripts/relayCompiler.js",
-    "sentry": "sh scripts/sentry.sh",
-    "start": "sh scripts/start.sh",
+    "sentry": "scripts/sentry.sh",
+    "start": "scripts/start.sh",
     "start:relay": "yarn relay && concurrently --raw --kill-others 'yarn relay --watch' 'yarn start'",
     "sync-schema": "rm -rf data && mkdir data && graphql-fetch-schema https://metaphysics-staging.artsy.net -o data",
-    "test": "sh scripts/test.sh",
```

Comment:
Do these updates mean the `.sh` files must be executable, or does `yarn` work either way?

Reply:
I think so. `yarn` would just run whatever command here, so I also `chmod +x` all the scripts in this PR.